<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btnTemp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/temp_btn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHumi">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>75</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/humidity_btn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSmoke">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/smoke_btn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAir">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>225</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/air_condition_btn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLamp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/lamp_btn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVideo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>375</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/camera_btn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCurtain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/curtain_btn.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOther">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>525</y>
     <width>151</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/other_dev_btn.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>535</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(115, 210, 22);
font: 11pt &quot;文泉驿微米黑&quot;;</string>
   </property>
   <property name="text">
    <string>报警设置</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>535</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(115, 210, 22);
font: 11pt &quot;文泉驿微米黑&quot;;</string>
   </property>
   <property name="text">
    <string>系统测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>535</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(115, 210, 22);
font: 11pt &quot;文泉驿微米黑&quot;;</string>
   </property>
   <property name="text">
    <string>系统简介</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>535</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(115, 210, 22);
font: 11pt &quot;文泉驿微米黑&quot;;</string>
   </property>
   <property name="text">
    <string>返回桌面</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>535</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(115, 210, 22);
font: 11pt &quot;文泉驿微米黑&quot;;</string>
   </property>
   <property name="text">
    <string>关于我们</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>535</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(115, 210, 22);
font: 11pt &quot;文泉驿微米黑&quot;;</string>
   </property>
   <property name="text">
    <string>退出系统</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>0</y>
     <width>871</width>
     <height>520</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>241</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿微米黑</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/img/running_text.png);
font: 11pt &quot;文泉驿微米黑&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>
  系统运行情况如下：
  成功打开的设备--&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>241</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿微米黑</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/img/alarming_text.png);
font: 11pt &quot;文泉驿微米黑&quot;;</string>
    </property>
    <property name="text">
     <string>
  系统报警情况如下：
  正在报警的设备--&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>241</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>文泉驿微米黑</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/img/server_text.png);
font: 11pt &quot;文泉驿微米黑&quot;;</string>
    </property>
    <property name="text">
     <string>
  服务器链接情况如下：
  --&gt;服务器链接成功</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>btnTemp_2</zorder>
   <zorder>btnHumi</zorder>
  </widget>
  <widget class="QPushButton" name="btnTemp_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/temp_btn_select.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHumi_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>75</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/huminity_btn_select.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSmoke_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/smoke_btn_select.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAir_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>225</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/air_condition_btn_select.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLamp_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/lamp_btn_select.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVideo_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>375</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/camera_btn_select.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCurtain_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>151</width>
     <height>74</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/curtain_btn_select.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOther_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>525</y>
     <width>151</width>
     <height>75</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/img/other_dev_btn_select.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>btnTemp_2</zorder>
  <zorder>btnOther_2</zorder>
  <zorder>btnCurtain_2</zorder>
  <zorder>btnVideo_2</zorder>
  <zorder>btnLamp_2</zorder>
  <zorder>btnAir_2</zorder>
  <zorder>btnSmoke_2</zorder>
  <zorder>btnHumi_2</zorder>
  <zorder>btnTemp</zorder>
  <zorder>btnHumi</zorder>
  <zorder>btnSmoke</zorder>
  <zorder>btnAir</zorder>
  <zorder>btnLamp</zorder>
  <zorder>btnVideo</zorder>
  <zorder>btnCurtain</zorder>
  <zorder>btnOther</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>pushButton_10</zorder>
  <zorder>pushButton_11</zorder>
  <zorder>pushButton_12</zorder>
  <zorder>pushButton_13</zorder>
  <zorder>pushButton_14</zorder>
  <zorder>frame</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
